xml/depl_maintenance.xml

```diff
@@ -419,14 +535,239 @@
       &compnode;s needs to be sufficient to evacuate the nodes one by one.
      </para>
     </listitem>
+    <listitem>
+     <para>
+      &o_img; needs to be set up with shared storage; that means the
+      <guimenu>Default Storage Store</guimenu> value in the &o_img; must
+      <emphasis>not</emphasis> be set to <literal>File</literal>.
```

Comment:
I'm a bit confused. The bug you are referring to is closed in https://github.com/SUSE-Cloud/doc-cloud/commit/05836e21 with the exact same wording as in this document.